docs/related-projects.md

```diff
@@ -16,7 +16,7 @@ We have six satellite apps that are maintained outside of the consumerfinance.go
 - [teachers-digital-platform](https://github.com/cfpb/teachers-digital-platform)
```

Comment:
I don't think I can let you get away with editing this file without correcting this part above 😄 TDP is no longer a satellite app, neither is ccdb5-ui, so we only have two satellite apps now, not 2.